client/ayon_houdini/plugins/create/create_generic.py

```diff
@@ -0,0 +1,735 @@
+            ),
+            hou.StringParmTemplate(
+                "AYON_creator_identifier", "Creator Identifier",
+                num_components=1,
+                default_value=(self.identifier,)
+            ),
+        ]:
+            if not parm_group.find(attribute.name()):
+                advanced_folder.addParmTemplate(attribute)
+
+        # Ensure all folders are up-to-date if they had previously existed
+        # already
+        for folder in [ayon_folder,
+                       attributes_folder,
+                       instance_attributes_folder,
+                       publish_attributes_folder,
+                       creator_attributes_folder,
+                       advanced_folder]:
+            if parm_group.find(folder.name()):
+                parm_group.replace(folder.name(), folder)  # replace
+        node.setParmTemplateGroup(parm_group)
+
+    def imprint(self,
+                created_instance: CreatedInstance,
+                values: dict,
+                update=False):
+
+        # Do not ever write these into the node.
+        values.pop("instance_node", None)
+        values.pop("instance_id", None)
+        values.pop("families", None)
+        if not values:
+            return
+
+        instance_node = hou.node(created_instance.get("instance_node"))
+
+        # Update attribute definition parms
+        self.create_attribute_def_parms(instance_node, created_instance)
+
+        # Creator attributes to parms
+        creator_attributes = values.pop("creator_attributes", {})
+        parm_values = {}
+        for attr, value in creator_attributes.items():
+            key = f"AYON_creator_attributes_{attr}"
+            parm_values[key] = value
+
+        # Publish attributes to parms
+        publish_attributes = values.pop("publish_attributes", {})
+        for plugin_name, plugin_attr_values in publish_attributes.items():
+            for attr, value in plugin_attr_values.items():
+                key = f"AYON_publish_attributes_{plugin_name}_{attr}"
+                parm_values[key] = value
+
+        # The remainder attributes are stored without any prefixes
+        # Prefix all values with `AYON_`
+        parm_values.update(
+            {f"AYON_{key}": value for key, value in values.items()}
+        )
+
+        set_values(instance_node, parm_values)
+
+        # TODO: Update defaults for Product Name
+        #   on the node so Houdini doesn't show them bold after save
+
+    def get_publish_families(self):
+        return [self.product_type]
+
+    def get_instance_attr_defs(self):
+        """get instance attribute definitions.
+
+        Attributes defined in this method are exposed in
+            publish tab in the publisher UI.
+        """
+
+        render_target_items = {
+            "local": "Local machine rendering",
+            "local_no_render": "Use existing frames (local)",
+            "farm": "Farm Rendering",
+        }
```

Comment:
Is it possible to add some condition here to add more items based on the instance ? 

I tried but I found out that it works per creator not per instance.

Reply:
No - unfortunately not. The current Creator system design _and_ the Publisher UI do not allow it unfortunately. @antirotor I believe is aware of it and it's part of the "Extended Publisher" epic.

Reply:
@MustafaJafar it may be that this PR https://github.com/ynput/ayon-core/pull/860 by @iLLiCiTiT may solve this. The question then becomes _WHAT_ would define which of the targets would be available?

Reply:
> is use existing frames (farm) not supported yet?

Render targets is not yet implemented for all product types.

Reply:
@BigRoy  I gave it a test and it actually works although it was hard for me to learn how to use it. I can't really tell if there are any alternative solutions but, we can discuss that on discord.
https://github.com/ynput/ayon-core/pull/860#pullrequestreview-2326349076

Reply:
Adding use exiting frames using farm is essential, there is a [dicussion](https://discord.com/channels/517362899170230292/517382145552154634/1287415312139943967) happened in couple of days ago, about this feature.
Local publish is tkaing huge amount of time and leaving houdini freezed until the publish get done. This a time saving for artists.